package.json

```diff
@@ -1,17 +1,20 @@
 {
-  "name": "Tanki",
+  "name": "tanki",
   "version": "1.0.0",
-  "description": "This is an awesome multiplayer game.",
-  "main": "index.js",
+  "description": "This is an awesome multi-player game.",
+  "main": "src/index.js",
```

Comment:
> This files doesn't actually exist yet… but it will soon 😄 

The `"main"` file is the one that will launch the server. We'll code it shortly.